.changeset/lovely-geckos-hammer.md

```diff
@@ -0,0 +1,5 @@
+---
+"@spectrum-css/link": minor
```

Comment:
Personally, I'd probably just call this a patch, since we categorized it as bug. I'll leave it up to you though.